Kernel/Prekernel/CMakeLists.txt

```diff
@@ -28,13 +35,20 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang$")
     target_link_libraries(${PREKERNEL_TARGET} PRIVATE clang_rt.builtins)
 endif()
 
+if ("${SERENITY_ARCH}" STREQUAL "x86_64")
+    set(ELF_OBJCOPY_TARGET "elf32-i386")
+elseif("${SERENITY_ARCH}" STREQUAL "aarch64")
+    message(SEND_ERROR "Prekernel is not needed on aarch64 and should not be compiled!")
+endif()
```

Comment:
I haven't looked at the full context, but do we just want to merge this into the existing "if x86_64 ... else"?

Reply:
That said, I do have to ask why we are even pulling this out into a separate variable. Just having `elf32-i386` inline should be fine, we already checked earlier that we aren't on a non-x86_64 architecture.